frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-summary.tsx

```diff
@@ -9,56 +9,46 @@ import { WINTOOLS_CONTAINER_NAMES } from '../modals/cdrom-vm-modal/constants';
 import { VMKind } from '../../types';
 import { V1Disk } from '../../types/vm/disk/V1Disk';
 
-const DiskSummaryRow: React.FC<DiskSummaryRowProps> = ({ title, value }) => (
-  <div className="kubevirt-disk-summary__disk">
-    <div id={`kubevirt-disk-summary-disk-title-${title}`}>{title}</div>
-    <div id={`kubevirt-disk-summary-disk-value-${value}`}>{value}</div>
-  </div>
-);
+import './disk-summary.scss';
 
 export const DiskSummary: React.FC<DiskSummaryProps> = ({ disks, vm }) => (
-  <div className="kubevirt-disk-summary">
-    {disks.map(({ name }, i) => {
+  <dl className="oc-vm-details__datalist kubevirt-disk-summary">
```

Comment:
I don't think you need the `.kubevirt-disk-summary` class right?

Reply:
it is used in the tests to identify this element ...

Reply:
could you just use `oc-vm-details__datalist` instead for the test?

Reply:
sure, it will require updating the tests, I would rather do it in a follow-up PR that address the tests.